### ISSUE TITLE
Verifying static image with cosign fails

### ISSUE THREAD

**Issue by priyawadhwa** [GoogleContainerTools/distroless#1233]
Keyless verification of the `gcr.io/distroless/static@sha256:0ef6d0699784a5d07eb2dedb05cab3471f7e86f0f519ee43edf9bb590f6bbf49` image fails with cosign:

```
$ cosign verify gcr.io/distroless/static@sha256:0ef6d0699784a5d07eb2dedb05cab3471f7e86f0f519ee43edf9bb590f6bbf49 --certificate-identity-regexp '.*' --certificate-oidc-issuer-regexp '.*'
Error: no matching signatures:
nil certificate provided
 checking expiry on certificate with bundle: certificate expired before signatures were entered in log: 2023-01-24T19:23:50Z is before 2023-01-24T11:23:53-08:00
main.go:63: error during command execution: no matching signatures:
nil certificate provided
 checking expiry on certificate with bundle: certificate expired before signatures were entered in log: 2023-01-24T19:23:50Z is before 2023-01-24T11:23:53-08:00
```
At a first glance the error message seems valid, it looks like the entry was added to the tlog about 3 seconds after the certificate expired:

Certificate Validity:
```
        Validity
            Not Before: Jan 24 19:13:50 2023 GMT
            Not After : Jan 24 19:23:50 2023 GMT
```

Rekor entry (I got this by checking the offline bundle in the signature manifest [here](https://console.cloud.google.com/gcr/images/distroless/global/static@sha256:64c21e821b8262fdf35f34e29abb21d33b8e25c99d192ad64e1e2debfcc192e4/details?tab=pull)):
```
$ rekor-cli get --log-index 11867074
LogID: c0d23d6ad406973f9559f3ba2d1ca01f84147d8ffc5b8445c224f98b9591801d
Index: 11867074
IntegratedTime: 2023-01-24T19:23:53Z
```

I'm not sure why there's a ten minute lag between creating the signature and adding the entry to the tlog, but maybe looking at build logs would be a first step? cc @loosebazooka lmk if you have any ideas!

**Comment by loosebazooka** [comment]
Yeah I think something about rekor changed, and keyless signing is having issues. I'm going to replace https://github.com/GoogleContainerTools/distroless/blob/main/cloudbuild_cosign.sh#L10 with individual signing and see if it works better

**Comment by priyawadhwa** [comment]
Thank you!!

**Comment by loosebazooka** [comment]
fixed by #1234
